lib/blather/client/client.rb

```diff
@@ -277,7 +277,7 @@ def ready!
 
     def client_post_init
       write_with_handler Stanza::Iq::Roster.new do |node|
-        roster.process node
+        roster.process(node) unless node.error?
```

Comment:
Right now we don't handle it, so it won't block this PR. I would be open to suggestions in a follow-up PR, but I don't have any any strong ideas about what I think we should do right now.